routers/api/v1/api.go

```diff
@@ -931,6 +930,13 @@ func Routes() *web.Route {
 			}, context.UserAssignmentAPI())
 		}, tokenRequiresScopes(auth_model.AccessTokenScopeCategoryUser), reqToken())
 
+		// Users SSH keys (publicly readable)
+		m.Group("/users", func() {
+			m.Group("/{username}", func() {
+				m.Get("/keys", user.ListPublicKeys)
```

Comment:
The content of `ListPublicKeys` need to be changed to remove private content.

Reply:
I've removed the key comment using the same method that's used in the `/username.keys` endpoint and rebased on top of `main`. I hope this is it but if I misunderstood please correct me!